[doc][ybm] Tables and Nodes tabs (#26137)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-cloud/cloud-monitor/monitor-nodes.md

---
title: Monitor YugabyteDB Aeon cluster nodes
headerTitle: Monitor nodes
linkTitle: Monitor nodes
description: Monitor the status of your nodes.
headcontent: Monitor the status of your nodes
menu:
  preview_yugabyte-cloud:
    identifier: monitor-nodes
    parent: cloud-monitor
    weight: 620
type: docs
---

> Review cluster nodes using the **Nodes** tab. The tab shows node status and activity, including the read and write operations, tablet distribution, memory and size, and which nodes host [Masters](../../../architecture/key-concepts/#master-server) and [TServers](../../../architecture/key-concepts/#tserver).

Reviewer: ```suggestion
Use the **Nodes** tab to review cluster nodes, including their status and activity. The tab displays read/write operations, tablet distribution, memory usage, size, and which nodes host [Masters](./../../architecture/key-concepts/#master-server) and [TServers](../../../architecture/key-concepts/#tserver).
```

What do you think about this?
